cbor-schema/plt.cddl

```diff
@@ -158,6 +158,17 @@ token-initialization-parameters = {
     ? "burnable": bool .default false
 }
 
+; Type alias for a 32-byte hash value
+hash = bytes .size(32)
+
+; Type alias for metadata with a URL and checksum
+metadata-url = {
+    ; A string field representing the URL
+    "value": string,
+    ; An optional 32-byte checksum value tied to the content of the URL
+    ? "checksum": hash
```

Comment:
I think this should be more explicit in how it's calculated. e.g. this is the SHA-256 hash of the URL contents.

Reply:
Resolving this, as the same discussion is taking place in https://github.com/Concordium/concordium-base/pull/648#discussion_r2100474810